### ISSUE TITLE
Update NVBit to 1.7

### ISSUE THREAD

**Issue by adofirst2018** [ParCoreLab/snoopie#6]
Have you plan  update NVBit to 1.7？ I try to update NVBit to 1.7[https://github.com/NVlabs/NVBit/releases/tag/1.7](url), But  meet some errors 
```
/work/Snoopie/src/mem_multigpu/mem_multigpu.cu(1234): error: class "ChannelDev" has no member "add_malloc"
     ctx_map_pair.second->channel_dev->add_malloc(ma);
```
I can not find add_malloc in core-1.7/utils/channel.hpp. But I am not  familiar with NVBIt, can you give me some advise?

**Comment by msasongko17** [comment]
We are actually working on an implementation of Snoopie that is based on NVIDIA compute sanitizer: https://docs.nvidia.com/compute-sanitizer/index.html . The problem with NVBit is that it is rarely updated. The last two versions are version 1.5.5 (released on February 3, 2022) and version 1.7 (released on July 12, 2024).

**Comment by adofirst2018** [comment]
@msasongko17 thank you for your reply.
